api/v1/verticadb_types.go

```diff
@@ -952,6 +971,12 @@ type SubclusterStatus struct {
 
 	// +operator-sdk:csv:customresourcedefinitions:type=status
 	Detail []VerticaDBPodStatus `json:"detail"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=status
+	// +optional
+	// State of the subcluster. true means the subcluster is currently down
+	// and must not be restarted.
+	Shutdown bool `json:"shutdown,omitempty"`
```

Comment:
I cannot see this field is updated in this PR.

Reply:
Because it is not needed so far. This PR is only for stopping/restarting a sandbox